src/System Application/App/Privacy Notice/src/SystemUpgradePrivacyNotices.Codeunit.al

```diff
@@ -34,14 +35,36 @@ codeunit 1567 "System Upgrade Privacy Notices"
         UpgradeTag.SetUpgradeTag(GetTeamsPrivacyNoticeUpgradeTag());
     end;
 
+    local procedure UpgradeMicrosoftLearnPrivacyNotice()
+    var
+        PrivacyNotice: Codeunit "Privacy Notice";
+        UpgradeTag: Codeunit "Upgrade Tag";
+        SystemPrivacyNoticeReg: Codeunit "System Privacy Notice Reg.";
+    begin
+        if UpgradeTag.HasUpgradeTag(GetMicrosoftLearnPrivacyNoticeUpgradeTag()) then
+            exit;
+
+        PrivacyNotice.CreateDefaultPrivacyNotices();
+
+        PrivacyNotice.SetApprovalState(SystemPrivacyNoticeReg.GetMicrosoftLearnID(), "Privacy Notice Approval State"::Agreed);
+
+        UpgradeTag.SetUpgradeTag(GetMicrosoftLearnPrivacyNoticeUpgradeTag());
+    end;
+
     [EventSubscriber(ObjectType::Codeunit, Codeunit::"Upgrade Tag", OnGetPerCompanyUpgradeTags, '', false, false)]
     local procedure RegisterPerCompanyTags(var PerCompanyUpgradeTags: List of [Code[250]])
     begin
         PerCompanyUpgradeTags.Add(GetTeamsPrivacyNoticeUpgradeTag());
+        PerCompanyUpgradeTags.Add(GetMicrosoftLearnPrivacyNoticeUpgradeTag());
     end;
```

Comment:
Privacy notice and privacy notice approval tables are set as `DataPerCompany = false`, so technically this could go into the per-database tags, but I don't think it matters that much?